otherlibs/stdune/path.ml

```diff
@@ -733,10 +733,11 @@ let is_root = function
   | In_source_tree s -> Local.is_root s
   | In_build_dir _ | External _ -> false
 
-let kind = function
-  | In_build_dir p -> Kind.append_local (Fdecl.get Build.build_dir) p
-  | In_source_tree s -> Kind.In_source_dir s
-  | External s -> Kind.External s
+let kind : t -> Outside_build_dir.t = function
```

Comment:
This function was weird with the old naming and it seems even weirder now. What does it do? Could you add a comment?

Reply:
It seems like it turns all paths, including those in the build dir, into a representation that's build-dir agnostic (or something like that). It would be nice to say it explicitly (and perhaps, also give it a less mysterious name).

Reply:
I renamed the function to something more useful this time.